CHANGELOG.md

```diff
@@ -4,12 +4,19 @@ Releases
 v1.13.0-dev (unreleased)
 -------------------
 
+-   Added a `yarpc.ClientConfig` interface to provide access to ClientConfigs.
+    All Dispatchers already implement this interface.
+-   Thrift: UberFx modules generated by the ThriftRW plugin now rely on
```

Comment:
In open-source land, it's just `fx`.

I think we'll need to be very picky about this distinction (`fx` is the framework, `uberfx` is the default bag of types) to avoid mass confusion.